charts/xui-webapp/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: xui-webapp
 home: https://github.com/hmcts/rpx-xui-webapp
-version: 0.2.9
+version: 0.2.10
```

Comment:
Intentional? This a recurring theme so I'm guessing something is happening during a `yarn install`. Can we clarify which version we should be on? If we need to stick to `0.2.9`, we need to fix it at that in the `package.json`.

Reply:
Jenkins automatically does this. I did not do. Its needed when it notices a change in Values.yaml